weblate/addons/forms.py

```diff
@@ -459,7 +459,9 @@ class AutoAddonForm(AutoForm, AddonFormMixin):
     def __init__(self, user, addon, instance=None, **kwargs):
         self.user = user
         self._addon = addon
-        super().__init__(obj=addon.instance.component, **kwargs)
+        # send addon instance as obj in case of a project addon
+        obj = addon.instance.component or addon.instance
+        super().__init__(obj=obj, **kwargs)
```

Comment:
`AutoForm` expects `obj` to be a `Component`, passing `AddOn` instead cannot work.